spec/models/item_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe Item, type: :model do
+  before do
+    @item = FactoryBot.build(:item)
+  end
+
+  describe '商品の投稿' do
+    context "商品の投稿ができる場合" do
+      it "正しい情報を入力すれば登録できる" do
+        expect(@item).to be_valid
+      end
+    end
+    context "商品の投稿ができない場合" do
+      it "画像がないと投稿できない" do
+        @item.images = nil
```

Comment:
```suggestion
        @item.image = nil
```
上記のように、こちらのimagesも単数形へ修正しましょう。

Reply:
修正しました。

Reply:
追加のテスト実行結果です。
[![Image from Gyazo](https://i.gyazo.com/6c70b575bd76a255564ccd84bf3cc86c.png)](https://gyazo.com/6c70b575bd76a255564ccd84bf3cc86c)